### PR TITLE
Couch SQL migration IV

### PR DESCRIPTION
**Pull request by snopoke**
@czue 
buddy @biyeun 

Getting close now I think

### CODE REVIEW

File: corehq/apps/couch_sql_migration/tests/test_migration.py [hunk 1]
```diff
@@ -146,8 +170,16 @@ def test_submission_error_log_migration(self):
         self._compare_diffs([])
 
     def test_hqsubmission_migration(self):
```
Comment (snopoke) [inline corehq/apps/couch_sql_migration/tests/test_migration.py]: Based on the git history, docs with type 'HQSubmission'
are just normal forms that were submitted by HQ ie. system forms
and were mostly used for:

* force closing cases: 37398d6
* submitting stock reports: 4f609c9, 23e729c

Comment (czue) [inline corehq/apps/couch_sql_migration/tests/test_migration.py]: how bout that.

Comment (czue) [inline corehq/apps/couch_sql_migration/tests/test_migration.py]: I guess the name makes sense and it's not a crazy idea (guessing we figured we would split them out in reports and stuff). too bad we were never consistent with it.

File: corehq/apps/couch_sql_migration/tests/test_migration.py [hunk 2]
```diff
@@ -216,18 +240,57 @@ def test_commit(self):
...
+
+    def _set_balance(self, balance, case_id, product_id):
+        from corehq.apps.commtrack.tests.util import get_single_balance_block
+        return self._submit_ledgers([
+            get_single_balance_block(case_id, product_id, balance)
+        ])
+
+    def test_migrate_ledgers(self):
+        case_id = uuid.uuid4().hex
+        create_and_save_a_case(self.domain_name, case_id=case_id, case_name="Simon's sweet shop")
```
Comment (czue) [inline corehq/apps/couch_sql_migration/tests/test_migration.py]: :candy:

**Comment by czue** [comment]
looks good - lots of real looking build failures

**Comment by biyeun** [comment]
build passes! merging

**Pull request merged**